core/src/main/scala/clue/package.scala

```diff
@@ -2,24 +2,35 @@
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
 import cats.Eq
+import cats.Functor
 import cats.MonadError
 import cats.effect.Concurrent
 import cats.effect.Deferred
+import cats.effect.MonadCancelThrow
 import cats.syntax.all.*
 import org.typelevel.log4cats.Logger
 
 package object clue {
   type FetchClient[F[_], S] = FetchClientWithPars[F, ?, S]
 
-  protected[clue] type Latch[F[_]] = Deferred[F, Either[Throwable, Unit]]
+  // None = canceled, Some(Right(())) = done, Some(Some(Left(t)) = error
```

Comment:
Commented code

Reply:
That's documentation.

Reply:
But it has an error, I'll fix and make it more explicit.